### PR TITLE
[OM-97588]: fix for tests done by Astra

### PR DESCRIPTION
**Pull request by libai98**
1) Failed creating policy binding:

[/Users/astra/VMTurbo/turbo-policy/config/samples]$ k apply -f policy_v1alpha1_policybinding.yaml
The PolicyBinding "policy-binding-containerverticalscale-sample" is invalid: spec.policyRef.kind: Unsupported value: "ContainerVerticalScale": supported values: "SLOHorizontalScale"

 ----- Added ContainerVerticalScale to be supported CRD.

2) Sample file need to be fixed:

[/Users/astra/VMTurbo/turbo-policy/config/samples]$ k apply -f policy_v1alpha1_containerverticalscale.yaml
The ContainerVerticalScale "containerverticalscale-sample" is invalid: spec.behavior.resize: Unsupported value: "manual": supported values: "Automatic", "Manual", "Recommend", "Disabled"

---- Added change to use " Manual" instead if "manual" in the sample file.
3) The binding sample file is missing for containerverticalcales:

[/Users/astra/VMTurbo/turbo-policy/config/samples]$ ls
./                                           policy_v1alpha1_containerverticalscale.yaml  policy_v1alpha1_slohorizontalscale.yaml
../                                          policy_v1alpha1_policybinding.yaml
The existing binding file policy_v1alpha1_policybinding.yaml is for horizontal scaling.

----added sample file for policy binding for containerverticalcales.

### CODE REVIEW

File: config/samples/policy_v1alpha1_containerverticalscale.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/part-of: turbo-policy
     app.kubernetes.io/managed-by: kustomize
     app.kubernetes.io/created-by: turbo-policy
-  name: containerverticalscale-sample
+  name: container-verticals-cale-sample
```
Comment (ading1977) [inline config/samples/policy_v1alpha1_containerverticalscale.yaml]: `container-verticals-cale-sample` -> `container-verticalscale-sample` ?

File: config/samples/policy_v1alpha1_policybinding_containerverticalscale.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# policy and the target services that the policy applies to
+apiVersion: policy.turbonomic.io/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: policy-binding-sample
```
Comment (ading1977) [inline config/samples/policy_v1alpha1_policybinding_containerverticalscale.yaml:5]: Shall we rename this? The other policy binding sample has the same name.

File: config/samples/policy_v1alpha1_containerverticalscale.yaml (hunk 1 above)
Comment (libai98) [reply config/samples/policy_v1alpha1_containerverticalscale.yaml]: Yes, it was changed. Thanks!

File: config/samples/policy_v1alpha1_policybinding_containerverticalscale.yaml (hunk 2 above)
Comment (libai98) [reply config/samples/policy_v1alpha1_policybinding_containerverticalscale.yaml]: Yes. Thanks!

**Pull request merged**